### PR TITLE
Use large runners

### PR DESCRIPTION
**Pull request by amessing-bdai**
Changelog:
- Change the github actions to use our large runners

### CODE REVIEW

File: .github/workflows/cd.yaml [hunk 1]
```diff
@@ -5,9 +5,14 @@ on:
     tags:
       - '*'
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```
Comment (mhidalgo-bdai) [inline .github/workflows/cd.yaml:9]: @amessing-bdai does this apply for tag push events?

**Review by mhidalgo-bdai — COMMENTED** [review]
Thanks for this Andrew!

File: .github/workflows/cd.yaml (hunk 1 above)
Comment (amessing-bdai) [reply .github/workflows/cd.yaml:9]: I believe `github.event.pull_request.number` is for PR and `github.ref` is for push

Comment (mhidalgo-bdai) [reply .github/workflows/cd.yaml:9]: Ah, I missed the `||` on first ready. Nevermind.

**Pull request merged**